testsuite/integration/microprofile/src/test/java/org/wildfly/test/integration/microprofile/jwt/propagation/JWTIdentityPropagationTestCase.java

```diff
@@ -372,16 +372,17 @@ public void setup(ManagementClient managementClient, String containerId) throws
         @Override
         public void tearDown(final ManagementClient managementClient, final String containerId) throws Exception {
             try (CLIWrapper cli = new CLIWrapper(true)) {
-                cli.sendLine(String.format("/subsystem=ejb3/application-security-domain=%s:remove()", EJB_SECURITY_DOMAIN));
-                cli.sendLine(String.format("/subsystem=elytron/security-domain=%s:remove()", EJB_SECURITY_DOMAIN));
-                cli.sendLine(String.format("/subsystem=elytron/filesystem-realm=%s:remove()", EJB_SECURITY_REALM));
-                cli.sendLine(String.format("/subsystem=elytron/security-domain=%s:remove()", ANOTHER_EJB_SECURITY_DOMAIN));
-                cli.sendLine(String.format("/subsystem=elytron/filesystem-realm=%s:remove()", ANOTHER_EJB_SECURITY_REALM));
                 cli.sendLine(String.format("/subsystem=elytron/virtual-security-domain=%s:remove()", SINGLE_DEPLOYMENT + ".ear"));
```

Comment:
Setup does this:

```
cli.sendLine(String.format("/subsystem=elytron/security-domain=%s:write-attribute(name=trusted-virtual-security-domains, value=[%s, %s])",
                        EJB_SECURITY_DOMAIN, SINGLE_DEPLOYMENT + ".ear", EAR_DEPLOYMENT_WITH_MP_JWT + ".ear")); 
```

That means EJB_SECURITY_DOMAIN has a reference to those two virtual-security-domain resources, which will prevent them being removed.

So I think this is needed before SINGLE_DEPLOYMENT or EAR_DEPLOYMENT_WITH_MP_JWT can be removed:

```
cli.sendLine(String.format("/subsystem=elytron/security-domain=%s:undefine-attribute(name=trusted-virtual-security-domains)", EJB_SECURITY_DOMAIN)); 
```

Reply:
done

Reply:
In the current push.  I have fixed a cli failure during tearDown.
The changes are primarily enabling logging of elytron and apache.  I added more detail to the failing assertion.  These changes will be removed in the future.